python/triton/language/core.py

```diff
@@ -701,12 +701,20 @@ def get_int_dtype(bitwidth: int, signed: bool) -> dtype:
         raise ValueError(f'Unsupported bitwidth {bitwidth} and signedness {signed}')
 
 
+class _value:
+    """Base class of values that exist in the triton IR (i.e. not constexprs).
```

Comment:
RE: https://github.com/triton-lang/triton/pull/4853#discussion_r1788002212

@Jokeren I hope this edit makes the class's purpose clearer.